modules/phpfpm/client.go

```diff
@@ -88,3 +109,34 @@ func (c client) fetchStatus(req *http.Request) (*status, error) {
 
 	return s, nil
 }
+
+func (c socketClient) Status() (*status, error) {
+	socket, err := fcgiclient.DialTimeout("unix", c.socket, c.timeout)
+	if err != nil {
+		return nil, fmt.Errorf("error on connecting to socket: %v", err)
+	}
+
```

Comment:
i think we need to `defer socket.Close()` right after `fcgiclient.DialTimeout` if there is no error